packages/sources/css/src/components/text-input/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@vtmn/css-text-input",
+  "version": "0.0.0",
```

Comment:
```suggestion
  "version": "0.0.1",
```
?

Reply:
No because lerna will do it for us automatically thanks to our conventional commits. But if I made a commit with "feat", he will choose "0.1.0"

Reply:
https://github.com/lerna/lerna/blob/main/commands/version/README.md#--conventional-commits